backend/cfg/cfg_comballoc.ml

```diff
@@ -38,7 +38,7 @@ let rec find_next_allocation : cell option -> allocation option =
         | Intop _ | Intop_imm _ | Intop_atomic _ | Floatop _ | Csel _
         | Reinterpret_cast _ | Static_cast _ | Probe_is_enabled _ | Opaque
         | Begin_region | End_region | Specific _ | Name_for_debugger _ | Dls_get
-        | Poll )
+        | Poll | Extcall _ )
```

Comment:
Should we comballoc across `Extcall`?

Reply:
(Discussed off-line, this is thought
to be fine because the callee should
not be able to see the uninitialized
bits of the "combined" allocation.)